src/Uno.UI.FluentTheme.v2/themeresources_v2.xaml

```diff
@@ -22634,9 +22634,9 @@
                 </VisualState>
               </VisualStateGroup>
             </VisualStateManager.VisualStateGroups>
-            <Path x:Name="LeftRadiusRenderArc" x:Load="False" Fill="{ThemeResource CardStrokeColorDefault}" VerticalAlignment="Bottom" Visibility="Collapsed" Margin="-4,0,0,0" Height="4" Width="4" Data="M4 0C4 1.19469 3.47624 2.26706 2.64582 3H0C1.65685 3 3 1.65685 3 0H4Z" />
+            <Path x:Name="LeftRadiusRenderArc" x:Load="False" Fill="{ThemeResource TabViewBorderBrush}" VerticalAlignment="Bottom" Visibility="Collapsed" Margin="-4,0,0,0" Height="4" Width="4" Data="M4 0C4 1.19469 3.47624 2.26706 2.64582 3H0C1.65685 3 3 1.65685 3 0H4Z" />
```

Comment:
This is not part of the change, I just realized is part of another PR that was merged last week that modified the TabView.xaml styles.